src/components/App/Pages/Portfolio/PortfolioBody/PortfolioEditSettings/test.js

```diff
@@ -13,7 +12,7 @@ test('PortfolioEditSettings', () => {
       },
     }
   })
-  const wrapper = shallow(<PortfolioEditSettings />)
-  expect(wrapper.find(MaterialButton).exists()).toBeTruthy()
+  const wrapper = mount(<PortfolioEditSettings />)
+  expect(wrapper.find('button').exists()).toBeTruthy()
```

Comment:
I wouldn't change this to mount if you can help it. Pretty high probability of finding some other `button` in the children that is not the one you were looking for.